### PR TITLE
MM-12505: remove all things WebRTC

### PR DESCRIPTION
**Pull request by lieut-data**
This complements https://github.com/mattermost/mattermost-server/pull/9700, https://github.com/mattermost/mattermost-webapp/pull/1907 and https://github.com/mattermost/mattermost-api-reference/pull/405.

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @lieut-data, noted - we can remove the docker hubs around the v5.6 release date.

File: source/guides/developer.rst [hunk 1]
```diff
@@ -2,3 +2,50 @@ Mattermost Developer's Guide
 ============================
 
 Detailed developer documentation is available at `https://developers.mattermost.com <https://developers.mattermost.com>`_.
+||||||| merged common ancestors
```
Comment (jasonblais) [inline source/guides/developer.rst]: Are these changes unintentional in developer.rst?

Comment (lieut-data) [reply source/guides/developer.rst]: Whoops, yes!
```suggestion
```

File: source/guides/developer.rst [hunk 2]
```diff
@@ -2,3 +2,49 @@ Mattermost Developer's Guide
 ============================
 
 Detailed developer documentation is available at `https://developers.mattermost.com <https://developers.mattermost.com>`_.
+Detailed information on contributing to the Mattermost open source project. For developer setup documentation `see developers.mattermost.com <https://developers.mattermost.com/contribute/server/developer-setup/>`_.
```
Comment (jasonblais) [inline source/guides/developer.rst]: Thanks @lieut-data. Are any of the changes to `../developer.rst` intentional?

Comment (lieut-data) [reply source/guides/developer.rst]: 🤦‍♂️ No, I'm just not paying attention here ;P Fixed!

**Comment by jasonblais** [comment]
@amyblais This is ready to merge, but propose we wait until the 5.6 release until we merge it.

I've submitted a PR to add a note at the top of the webrtc.rst doc announcing the removal and linking to the forum post for more info https://github.com/mattermost/docs/pull/2359

**Pull request merged**